aksel.nav.no/website/pages/templates/soknad-introside/intro-med-alternativ-soknad.tsx

```diff
@@ -0,0 +1,238 @@
+                  <i>
+                    Du må ha meldt deg som arbeidssøker før du kan søke om
+                    dagpenger.
+                  </i>
+                </List.Item>
+                <List.Item>
+                  Dokumentasjon brukeren kan bli bedt om.{" "}
+                  <i>
+                    Noen av opplysningene du gir underveis vil du bli bedt om å
+                    dokumentere. Du vil trenge xx og xx for å fullføre denne
+                    søknaden.
+                  </i>
+                </List.Item>
+                <List.Item>
+                  Automatisk lagring.{" "}
+                  <i>
+                    Vi lagrer svarene dine (xx timer) mens du fyller ut, så du
+                    kan ta pauser underveis.
+                  </i>
+                </List.Item>
+                <List.Item>
+                  Antall steg og estimert tidsbruk.{" "}
+                  <i>
+                    Det er XX steg i søknaden, og du kan regne med å bruke ca.
+                    XX minutter.
+                  </i>
+                </List.Item>
+                <List.Item>
+                  Søknadsfrister.{" "}
+                  <i>Husk at du må søke om xx innen xx dager.</i>
+                </List.Item>
+                <List.Item>
+                  Saksbehandlingstider og info om gyldighet, krav osv.{" "}
+                  <i>
+                    Vi bruker ca. xx uker på å behandle søknaden din. Husk at du
+                    må sende meldekort xx ofte selv om du ikke har fått svar på
+                    søknaden din om dagpenger ennå.
+                  </i>
+                </List.Item>
+              </List>
+            </Box>
+            <Box>
+              <Accordion>
+                <Accordion.Item>
+                  <Accordion.Header>
+                    Informasjon vi henter om deg
+                  </Accordion.Header>
+                  <Accordion.Content>
+                    <BodyLong spacing>
+                      Her skal det så informasjon om hvor vi vil hente
+                      opplysninger om søkeren og hva slags opplysninger vi
+                      henter.
+                    </BodyLong>
+                  </Accordion.Content>
+                </Accordion.Item>
+                <Accordion.Item>
+                  <Accordion.Header>
+                    Hvordan vi behandler personopplysninger
+                  </Accordion.Header>
+                  <Accordion.Content>
+                    <BodyLong spacing>
+                      Her skal det stå informasjon om hvordan vi behandler
+                      personopplysningene til søkeren.
+                    </BodyLong>
+                  </Accordion.Content>
+                </Accordion.Item>
+                <Accordion.Item>
+                  <Accordion.Header>Automatis behandling</Accordion.Header>
+                  <Accordion.Content>
+                    <BodyLong spacing>
```

Comment:
Bytt teksten i accordion item med dette:
Her skal det stå informasjon om hva automatisk behandling er, hva det betyr for søkeren og informasjon om søkerens rettigheter ved automatisk avslag.